api_docs/changelog.md

```diff
@@ -20,6 +20,14 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 248**
```

Comment:
You would also need to bump the `API_FEATURE_LEVEL` field in `version.py`.

Reply:
Updated in `version.py` and made necessary changes api changelog and docs.